app/assets/javascripts/application.js

```diff
@@ -4,3 +4,8 @@
 //= require bootstrap
 //= require bootstrap-notify
 //= require_tree .
+//= require clipboard
+
+$(document).ready(function() {
+  new Clipboard('#clip');
```

Comment:
Yes, it's currently needed in one place only, but I will change it to class regardless.